### PR TITLE
Integrating test data prediction pipeline

### LINKED ISSUE #45: Pipeline: Run against test data
We need our pipeline to also be able to run against test data.

### PR DESCRIPTION
**Pull request by ASAAR**
This PR closes #45 and #104, includes the following,
- trained model is used to make predictions on test data
- workers continue to request tickets until tickets are complete
- hub saves results from all tickets into csv files for submission

### CODE REVIEW

**Comment by ZakeryFyke** [comment]
I'll fix your failed tests. -_-
(Thanks btw)

File: modules/DMZ/utils/config_options.py [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-        self.prediction_type = None
-        self.target_column = None
-
-        self.get_config_options()
-        self.validate_options()
+        self.training_file_name = "titanic_train.csv"
+        self.test_file_name = "titanic_test.csv"
+        self.prediction_type = "classification"
+        self.target_column = "Survived"
+        self.id_column = "PassengerId"
```
Comment (ASAAR) [inline modules/DMZ/utils/config_options.py]: Really just the values right now, I got sick of typing them in lol

File: modules/worker/gym.py [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
     def validate_model(self, data_control, model_control):
-        trained_model = model_control.model[1].fit(data_control.validation_pack.x_train,
+        self.trained_model = model_control.model[1].fit(data_control.validation_pack.x_train,
                                                    data_control.validation_pack.y_train)
 
-        self.score = trained_model.score(data_control.validation_pack.x_test,
+        self.score = self.trained_model.score(data_control.validation_pack.x_test,
                                          data_control.validation_pack.y_test)
 
         print(self.score)
```
Comment (asclines) [inline modules/worker/gym.py]: Should this be here?

**Comment by asclines** [comment]
How much testing has been done on the new methods?

**Comment by ASAAR** [comment]
Testing is limited currently, we are waiting on configuration tools

**Pull request merged**